OpenEdXMobile/src/main/java/org/edx/mobile/http/constants/TimeInterval.java

```diff
@@ -0,0 +1,48 @@
+package org.edx.mobile.http.constants;
+
+/**
+ * Constants for time intervals in seconds. These are meant to be used for defining the maximum ages
+ * in the Cache-Control headers that are defined in Retrofit's HTTP API interfaces.
+ */
+public class TimeInterval {
```

Comment:
The Java framework does provide the `TimeUnit` utility class, which provides static methods to convert between time units (though it doesn't have units greater than a day). However, only constant values can be provided as annotation values to be used in Retrofit method definitions, so this utility class has been defined to provide the units as constants.